homeassistant/components/shopping_list/config_flow.py

```diff
@@ -0,0 +1,36 @@
+"""Config flow to configure ShoppingList component."""
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+
+class ShoppingListFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for ShoppingList component."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+    SHOPPING_LIST = "Shopping List"
+
+    def __init__(self):
+        """Init ShoppingListFlowHandler."""
+        self._errors = {}
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        self._errors = {}
```

Comment:
Not modified

Reply:
What does that mean?

Reply:
self._errors = {} is initialized but not modified.

So you can remove the var.